crates/key-derivation/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+edition = "2021"
+name = "strata-key-derivation"
+version = "0.1.0"
+
+[lints]
+rust.rust_2018_idioms = { level = "deny", priority = -1 }
+rust.unused_crate_dependencies = "deny"
+rust.unused_must_use = "deny"
+
+[dependencies]
+bitcoin = { workspace = true, features = ["rand-std"] }
+thiserror.workspace = true
+zeroize = { workspace = true, optional = true }
+
+[features]
+default = ["zeroize"]
+zeroize = ["zeroize/derive"]
```

Comment:
Why do you need the `derive` feature? You're doing manual trait implementations.

Reply:
Yes, good catch!
I don't need `derive`!

Reply:
Done in ad1128908cc6b7c78c8962da4006f10950dc7b52